content/docs/components/serving/seldon.md

```diff
@@ -10,7 +10,7 @@ Seldon Core comes installed with Kubeflow. The [Seldon Core documentation site](
 
 If you have a saved model in a PersistentVolume (PV), Google Cloud Storage bucket or Amazon S3 Storage you can use one of the [prepackaged model servers provided by Seldon Core](https://docs.seldon.io/projects/seldon-core/en/latest/servers/overview.html).
 
-Seldon Core also provides [language specific model wrappers](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/language_wrappers.html) to wrap your inference code for it to run in Seldon Core.
+Seldon Core also provides [language specific model wrappers](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/README.html) to wrap your inference code for it to run in Seldon Core.
```

Comment:
I think you need to revert this change. The README page gives a "page not found" error. A recent PR fixed the link.